src/relentless/model/potential/__init__.py

```diff
@@ -38,6 +38,15 @@
     HarmonicAngle
     HarmonicCosineAngle
 
+Dihedral potentials
+===================
+
+.. autosummary::
+    :toctree: generated/
+
+    OPLSDihedral
+    RyckaertBellemansDihedral
+    DihedralSpline
```

Comment:
I noticed we are inconsistent between bond, angle, and dihedral in how we are ordering these. Should they all be alphabetical, or logically ordered?

Reply:
I reordered these to be alphabetic which probably makes the most sense when searching for something.